docs/protocol/network/messages/peer-discovery.md

```diff
@@ -0,0 +1,30 @@
+# Peer discovery
+
+Peer discovery protocol allows nodes to exchange information of known peers to other nodes. This information is usually used by peer discovery algorithms.
```

Comment:
"Usually" seems ambiguous. Maybe even ominous.

Reply:
Ok, I removed it. I left it as: "This information may be used by peer discovery algorithms".